### PR TITLE
Use `expect_issue` in `Style` specs

### PR DESCRIPTION
**Pull request by FnControlOption**
I'm not sure what to do with this:

```cr
expect_issue rule, <<-CRYSTAL, name: name
    %{name} = #{value}
  # ^{name} error: Constant name should be screaming-cased: #{expected}, not #{name}
  CRYSTAL
```

### CODE REVIEW

**Comment by veelenga** [comment]
What's wrong with the example in the description? Hard to read?

File: src/ameba/ast/util.cr [hunk 1]
```diff
@@ -158,4 +158,16 @@ module Ameba::AST::Util
...
+  # Returns the exp code of a control expression.
+  # Wraps implicit tuple literal with curly brackets (e.g. multi-return).
+  def control_exp_code(source, node : Crystal::ControlExpression)
+    return unless (exp = node.exp)
+    return unless (exp_code = node_source(exp, source.lines))
+    return exp_code unless exp.is_a?(Crystal::TupleLiteral) && exp_code[0] != '{'
+    return unless (exp_start = exp.elements.first.location)
+    return unless (exp_end = exp.end_location)
+
+    "{#{source_between(exp_start, exp_end, source.lines)}}"
```
Comment (veelenga) [inline src/ameba/ast/util.cr:171]: isn't this the same?

```suggestion
    source_between(exp_start, exp_end, source.lines).to_s
```

File: src/ameba/rule/style/variable_names.cr [hunk 2]
```diff
@@ -35,6 +35,18 @@ module Ameba::Rule::Style
       issue_for node, MSG % {expected, node.name}
     end
 
+    # TODO: Handle special case where instance variable is method parameter.
```
Comment (veelenga) [inline src/ameba/rule/style/variable_names.cr:38]: i think there is no proper way to handle this without changing crystal internals. This was discussed and tried several times

Comment (FnControlOption) [reply src/ameba/rule/style/variable_names.cr:38]: I'm thinking create a NodeVisitor subclass that keeps track of the `location` of variables and skips each method parameter that is at the same `location` as an instance variable.

Comment (veelenga) [reply src/ameba/rule/style/variable_names.cr]: Oh, sorry, I confused this with a different issue.

File: src/ameba/ast/util.cr [hunk 3]
```diff
@@ -158,4 +158,16 @@ module Ameba::AST::Util
       false
     end
   end
+
+  # Returns the exp code of a control expression.
+  # Wraps implicit tuple literal with curly brackets (e.g. multi-return).
+  def control_exp_code(source, node : Crystal::ControlExpression)
```
Comment (veelenga) [inline src/ameba/ast/util.cr:164]: could you please cover this method by unit tests

File: spec/ameba/rule/style/variable_names_spec.cr [hunk 4]
```diff
@@ -25,25 +27,30 @@ module Ameba
...
 
-    it_reports_var_name %(
-      class Greeting
-        def initialize(@badNamed = nil)
+    it "reports instance variable name" do
+      expect_issue subject, <<-CRYSTAL
+        class Greeting
+          def initialize(@badNamed = nil)
+                       # ^ error: Var name should be underscore-cased: @bad_named, not @badNamed
+          end
```
Comment (veelenga) [inline spec/ameba/rule/style/variable_names_spec.cr:41]: could you please also add a test for a method with multiple instance vars

**Review by veelenga — APPROVED** [review]

**Review by Sija — APPROVED** [review]

**Comment by Sija** [comment]
Awesome, thanks @FnControlOption! ❤️

**Pull request merged**